src/main/java/Parser.java

```diff
@@ -0,0 +1,50 @@
+public class Parser {
+
+    public static String parseInput(String command) {
+        return command.toLowerCase().trim();
+    }
+
+    public static String[] parseTask(String command, Action taskType) {
+        String[] parameters = new String[2];
+        switch (taskType) {
+        case TO_DO:
+            parameters[0] = command.substring(command.indexOf("todo") + 5);
```

Comment:
Another way to realise the task: using String[] parameters = command.split("……");